README.mediawiki

```diff
@@ -1295,6 +1295,13 @@ Those proposing changes should consider that ultimately consent may rest with th
 | Gloria Zhao
 | Informational
 | Draft
+|-
+| [[bip-0442.md|442]]
+| Consensus (soft fork)
+| OP_PAIRCOMMIT
+| moonsettler
```

Comment:
The CI issue is caused by Brandon missing here, because now the authors don’t match up between the BIP and the table.

```suggestion
| moonsettler, Brandon Black
```

Reply:
looking at that `buildtable.pl` script... should we even use `<pre>` in markdown?

Reply:
Ah, yes, we fixed the formatting checker for MarkDown files, because in MarkDown BIPs the email addresses of the authors didn’t render: https://github.com/bitcoin/bips/pull/1759

Please use <code>```</code> before and after the preamble now!